web/impact/impact/permissions/graphql_permissions.py

```diff
@@ -0,0 +1,43 @@
+from accelerator.models import (
+    UserRole,
+)
+from accelerator_abstract.models.base_user_utils import is_employee
+from accelerator.models import ACTIVE_PROGRAM_STATUS
+
+
+def visible_roles(current_user):
```

Comment:
This function is longer than our standards call for. 
In future, consider taking the following steps to get functions under 10 lines

- declare constants outside of functions (this is a benefit, since it allows them to be referred to in tests)
- extract complex queries to helper functions (allowing them to be re-used, and also granting them a helpful name)